server/controllers/stock/reports/stock/exit_patient_receipt.js

```diff
@@ -38,6 +38,7 @@ function stockExitPatientReceipt(req, res, next) {
     JOIN patient p ON p.uuid = m.entity_uuid
     JOIN project proj ON proj.id = p.project_id
     JOIN user u ON u.id = m.user_id
+    JOIN document_map dm ON dm.uuid = m.document_uuid 
```

Comment:
Personally, I worry this will have a larger performance impact than you expect, for no real gain in functionality.  The table `document_map` has ever single reference in the system.  That may make the server work really hard ... an SQL performance analysis would be needed to see exactly how it composes this query.

We'll have to keep an eye on this in the future to make sure it doesn't slow down execution.

Reply:
👍 you're right